setup.py

```diff
@@ -6,45 +6,31 @@
 here = path.abspath(path.dirname(__file__))
 
 # Get the long description from the README file
-with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+with open(path.join(here, "README.md"), encoding="utf-8") as f:
     long_description = f.read()
-released_version = os.environ.get('VERSION')
+released_version = "1.9.2dev171"
```

Comment:
please revert this

Reply:
It kept producing "already existing" versions, so I had to hard code it. Obviously I won't be deploying with that dev version.